### PR TITLE
locator-info: update script with new stow format

### PR DESCRIPTION
**Pull request by hagai-weka**

### CODE REVIEW

File: topics/snapshots/locator-info/src/stow.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ class _WekaCompatabilityRange:
     6: _WekaCompatabilityRange("v3.12.0"),
     7: _WekaCompatabilityRange("v3.13.0"),
     8: _WekaCompatabilityRange("v3.14.0"),
+    9: _WekaCompatabilityRange("v3.15.0"),
```
Comment (EyalIO) [inline topics/snapshots/locator-info/src/stow.py:28]: will we update this when it becomes v4.0 ?

Comment (hagai-weka) [reply topics/snapshots/locator-info/src/stow.py:28]: good question. when is the transition supposed to happen?
(if it was up to me, i'd say it's too late, and make 4.0 the next version after 3.15 . just saying)

Comment (EyalIO) [reply topics/snapshots/locator-info/src/stow.py:28]: I think that's what's being done -- except we're never going to release 3.15 :-)

Comment (hagai-weka) [reply topics/snapshots/locator-info/src/stow.py:28]: oh. sneaky. so technically this line isn't incorrect :P

**Pull request merged**